dss/operations/checkout.py

```diff
@@ -63,72 +62,72 @@ def _sleepy_checkout(self, function=None, **kwargs):
+class Remove(CheckoutHandler):
     def process_keys(self):
-        """
-        Remove a bundle from the checkout bucket
-        """
+        """Remove keys from the checkout bucket."""
         for _key in self.keys:
             if DSS_BUNDLE_KEY_REGEX.match(_key):
-                for key in self.handle.list(self.bucket, _key):  # handles checkout/bundle/*
-                    self._verify_delete(self.handle, self.bucket, key)
+                for key in self.handle.list(self.checkout_bucket, _key):  # handles checkout/bundle/*
+                    self._verify_delete(self.handle, self.checkout_bucket, key)
                 uuid, version = self._parse_key(_key)
                 manifest = get_bundle_manifest(replica=self.replica, uuid=uuid, version=version)
                 if manifest is None:
-                    logger.warning(f"unable to locate manifest for fqid: {self.bucket}/{_key}")
+                    logger.warning(f"Unable to locate manifest for fqid: {self.checkout_bucket}/{_key}")
                     continue
                 for _files in manifest['files']:
                     key = compose_blob_key(_files)
-                    self._verify_delete(self.handle, self.bucket, key)
+                    self._verify_delete(self.handle, self.checkout_bucket, key)
             elif _key.startswith(FILE_PREFIX):
                 # should handle other keys, files/blobs
-                file_metadata = self._verify_get(self.handle, self.replica.bucket, _key)
-                self._verify_delete(self.handle, self.bucket, key=compose_blob_key(file_metadata))
+                file_metadata = self._get_metadata(self.handle, self.replica.bucket, _key)
+                self._verify_delete(self.handle, self.checkout_bucket, key=compose_blob_key(file_metadata))
+            else:
+                print(f'INVALID KEY REGEX: {_key}')
 
 
 @checkout.action("verify",
                  arguments={"--replica": dict(choices=[r.name for r in Replica], required=True),
-                            "--keys": dict(nargs="+", help="keys to check the status of", required=True)})
-class verify(CheckoutHandler):
+                            "--keys": dict(nargs="+", help="Keys to check the status of.", required=True)})
+class Verify(CheckoutHandler):
     def process_keys(self):
-        """
-        Verify that keys are in the checkout bucket
-        """
+        """Verify that keys are in the checkout bucket."""
         checkout_status = dict(replica=self.replica.name)
         for _key in self.keys:
             if DSS_BUNDLE_KEY_REGEX.match(_key):  # handles bundles/fqid keys or fqid
                 uuid, version = self._parse_key(_key)
                 bundle_manifest = get_bundle_manifest(replica=self.replica, uuid=uuid, version=version)
-                checkout_bundle_contents = [x[0] for x in self.handle.list_v2(bucket=self.bucket,
+                print(bundle_manifest)
```

Comment:
i would avoid printing stuff that does not need to be printed, unless you can mute it, If i want to pipe this into JQ, the print statement  causes problems.

Reply:
Ah, this was cruft.  Should be removed now.